test/DocumentFormat.OpenXml.Tests/Linq/W.cs

```diff
@@ -0,0 +1,842 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Xml.Linq;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+namespace DocumentFormat.OpenXml.Tests.Linq
```

Comment:
Cool - let's keep this here for now and we can merge it into the main project at some point

Reply:
Yes, I needed to retain that here for unit testing.